README.md

```diff
@@ -1,2 +1,175 @@
-# awesome-pieces
-An awesome &amp; curated list of best applications and tools with Pieces.
+# Awesome Pieces <img src="awesome.svg" text="Awesome">
+
+[<img src="logonew.png" align="right" width="75">](https://pieces.app/)
```

Comment:
@anushka-23g looks like this image isn't coming through correctly - just some formatting in the markdown that needs adjusted

Reply:
its showing up on your branch when i open it but in the rich diff it is not coming through. interesting

Reply:
im going to approve and test this when it is merged because it seems like it is only in the rich diff